helpers/integration.go

```diff
@@ -97,13 +132,25 @@ func (t *TaskRun) GetTestResult() (*HACBSTestResult, error) {
 	if t.testResult != nil {
 		return t.testResult, nil
 	}
+	// load schema for test validation
+	sch, err := jsonschema.CompileString("schema.json", testSchema)
+	if err != nil {
+		return nil, fmt.Errorf("error while compiling json data for schema validation: %w", err)
+	}
 
 	for _, taskRunResult := range t.trStatus.TaskRunResults {
 		if taskRunResult.Name == HACBSTestOutputName {
 			var result HACBSTestResult
+			var v interface{}
 			err := json.Unmarshal([]byte(taskRunResult.Value.StringVal), &result)
 			if err != nil {
-				return nil, err
+				return nil, fmt.Errorf("error while mapping json data from taskRun %s: to HACBSTestResult - %w", taskRunResult.Name, err)
+			}
+			if err := json.Unmarshal([]byte(taskRunResult.Value.StringVal), &v); err != nil {
+				return nil, fmt.Errorf("error while mapping json data from taskRun %s: - %w", taskRunResult.Name, err)
```

Comment:
Why `-` before `%w` ?

Reply:
This was helpful for me visually when I was tracing back some errors but can change back to follow prior formatting convention.